data/data-pipeline/data_pipeline/application.py

```diff
@@ -105,10 +105,18 @@ def score_full_run():
 
 
 @cli.command(help="Generate Geojson files with scores baked in")
-def geo_score():
+@click.option("-ds", "--data-source", required=False, type=str)
+def geo_score(datasource: str):
     """CLI command to generate the score"""
 
-    score_geo()
+    if datasource is "" or " ":
+        score_geo("local")
+    if not datasource:
+        score_geo("local")
+    if datasource is "aws":
+        score_geo("aws")
+    else:
+        logger.warn('This data source is unimplemented, %s' % datasource)
 
```

Comment:
this feels like the worst python code ever written 😂

Reply:
cc: @esfoobar-usds 😂

Reply:
https://github.com/usds/justice40-tool/pull/559/commits/7a943b749fa773bda830ae7374e6d88bb3a83e5f cleaned it up a bit cause empty string is apparently falsey like JavaScript and used `elif`

Reply:
I would use Click.Choice like so:

```
@click.option('--data-source;, type=click.Choice(['local', 'staging', 'production'], default='local')
```

Then you can pass `data_source` to the runner.

@vim-usds and I discussed having similar options for the client frontend.